meson_options.txt

```diff
@@ -17,6 +17,7 @@ option('plugin_synaptics', type: 'boolean', value: true, description : 'enable S
 option('plugin_thunderbolt', type : 'boolean', value : true, description : 'enable Thunderbolt support')
 option('plugin_redfish', type : 'boolean', value : true, description : 'enable Redfish support')
 option('plugin_uefi_capsule', type : 'boolean', value : true, description : 'enable UEFI support')
```

Comment:
just realized forgot the word capsule in this description too.

Reply:
Fixed as an additional commit, thanks.